README.rst

```diff
@@ -28,133 +28,43 @@ Nengo: Large-scale brain modelling in Python
 Installation
 ============
 
-To install Nengo, use::
+To install Nengo::
 
   pip install nengo
 
 Nengo depends on `NumPy <http://www.numpy.org/>`_.
-If you have difficulty installing,
-try installing NumPy first (see below).
+If you have difficulty installing Nengo, try
+`installing NumPy <https://pythonhosted.org/nengo/getting_started.html#installation>`_ first.
 
 Nengo supports Python 2.6, 2.7, and 3.3+.
 
-Developer install
------------------
-
-.. code-block:: bash
-
-   git clone https://github.com/nengo/nengo.git
-   cd nengo
-   python setup.py develop --user
-
-If you're using a ``virtualenv``
-(recommended!) then you can omit the ``--user`` flag.
-
-Installing requirements
------------------------
-
-Nengo's main requirement is Numpy
-(and optionally Scipy for better performance).
-Currently, installing Numpy from ``pip`` will install
-an unoptimized (slow) version,
-so it is preferable to install Numpy
-using one of the following methods.
-It is important to install Numpy
-before installing the other requirements.
-
-Anaconda
-^^^^^^^^
-
-Numpy is included as part of the
-[Anaconda](https://store.continuum.io/cshop/anaconda/)
-Python distribution.
-This is a straightforward solution to get Numpy working on
-Windows, Mac, or Linux.
-
-Ubuntu
-^^^^^^
-
-On Ubuntu and derivatives (e.g. Linux Mint),
-Numpy and Scipy can be installed using ``apt-get``:
-
-.. code-block:: bash
-
-   sudo apt-get install python-numpy python-scipy
-
-From source
-^^^^^^^^^^^
-
-Numpy can be installed from source.
-This is the most complicated method,
-but is also the most flexible
-and results in the best performance.
-See the detailed instructions
-[here](http://hunseblog.wordpress.com/2014/09/15/installing-numpy-and-openblas/).
-
-Other requirements
-^^^^^^^^^^^^^^^^^^
-
-To install optional requirements to enable additional features, do
-
-.. code-block:: bash
-
-   pip install -r requirements.txt
-   pip install -r requirements-optional.txt
-
-The testing and documentation requirements
-can be found in similarly named files.
-
-Documentation & Examples
-========================
-
-Documentation and examples can be found at
-`<https://pythonhosted.org/nengo/>`_.
-
-
-Running tests
+Examples
+========
+
+Here are five of
+`many examples <https://pythonhosted.org/nengo/examples.html>`_
+showing how Nengo enables the creation and simulation of
+large-scale neural models in few lines of code.
+
+1. `100 LIF neurons representing a sine wave
```

Comment:
I might put the NEF summary notebook in this list, too, though I like having the many-neurons example first, since it's a clear exposition of basic Nengo syntax.